src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruit, int quantity) {
+        Storage.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return Storage.storage.get(fruit);
```

Comment:
There's a potential `NullPointerException` if the `fruit` does not exist in the storage. You should check if the storage contains the key before trying to get its value. If the fruit is not present, you may want to return a default value, such as 0.